modules/programs/mise.nix

```diff
@@ -108,9 +108,9 @@ in {
         eval "$(${getExe cfg.package} activate bash)"
       '';
 
-      zsh.initExtra = mkIf cfg.enableZshIntegration ''
+      zsh.initContent = mkIf cfg.enableZshIntegration (lib.mkOrder 1200 ''
```

Comment:
why the change here ?

Reply:
What do you mean?

Reply:
has yes I thought it was 1000 `(lib.mkIf (cfg.initExtra != "") (mkOrder 1200 cfg.initExtra))`

Reply:
> has yes I thought it was 1000 `(lib.mkIf (cfg.initExtra != "") (mkOrder 1200 cfg.initExtra))`

We were saying above it would be nice to if `initExtra` got shifted down to `mkOrder 1000` though. Would make it less cumbersome to use for shifting from initExtra to initContent.

Reply:
yes that would make the transition easier/clearer